### ISSUE TITLE
Support for ARM ?

### ISSUE THREAD

**Issue by stuartly** [nccgroup/TriforceAFL#6]
I want to use TriforceAFL to fuzz a arm binary, does it support ?

**Comment by jhertz** [comment]
yes it does. checkout the arm32 branch: https://github.com/nccgroup/TriforceAFL/tree/arm32
